packages/forma-36-react-components/src/components/EditorToolbar/EditorToolbar/README.mdx

```diff
@@ -0,0 +1,57 @@
+import { Button } from '../../Button';
+import { EditorToolbar } from './EditorToolbar';
+import { EditorToolbarButton } from '../EditorToolbarButton';
+
+# EditorToolbar
+
+The EditorToolbar is a toolbar component for rich text editors. It provides a styled shell that toolbar actions such as buttons can be wrapped in.
+
+<EditorToolbar style={{ justifyContent: 'space-between' }}>
```

Comment:
This part should be displayed below ## Example of usage section?

Reply:
Do we want to take advantage of MDX and render the component? That was my thinking here, and then show actual code in examples of usage.